curs_main.c

```diff
@@ -50,6 +50,7 @@
 #include "protos.h"
 #include "sort.h"
 #include "thread.h"
+#include <term.h>
```

Comment:
I don't understand. `tigetstr()` is, if I read the code, used unconditionally, so the header file needs to be included unconditionally as well.
Addiitonally, on NetBSD, neither of the defines is defined–we're using curses there, not ncurses.